securedrop/tests/test_crypto_util.py

```diff
@@ -19,6 +19,15 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
+    def test_word_lists_imports(self):
```

Comment:
Can we name this something more precise, e.g. `test_word_list_does_not_contain_empty_strings`